apps/web/app/(authenticated)/(app)/onboarding/client.tsx

```diff
@@ -43,15 +45,30 @@ export const Onboarding: React.FC<Props> = ({ tenantId }) => {
     },
     onError(err) {
       console.error(err);
-      toast({ title: "Error", description: err.message, variant: "destructive" });
+      toast({
+        title: "Error",
+        description: err.message,
+        variant: "destructive",
+      });
     },
   });
-
+  const { theme } = useTheme();
   return (
-    <div className="flex items-center justify-center w-full min-h-screen">
-      <Card>
+    <div className="flex flex-col justify-center items-center w-full min-h-screen">
```

Comment:
![CleanShot 2023-07-21 at 17 32 16](https://github.com/unkeyed/unkey/assets/18246773/a1b23314-5324-447a-89c3-d4ec94b80083)
This doesn't seem to use darkmode?

also again, let's use less gradients for now

Reply:
oh this weird should have worked will fix it